segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/AclMap.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.server.host;
+
+import io.pravega.segmentstore.storage.impl.extendeds3.AclSize;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+/**
+ * This class represents the ACL to object key mapping for S3 wrapper for integration tests.
+ */
+public class AclMap {
+    private ConcurrentMap<String, AclSize> aclMap = new ConcurrentHashMap<>();
```

Comment:
@andreipaduroiu The introduction of this class does not address my concerns. I just did not catch it in the code review. 
@arvindkandhare Please change the the test code so that there is not shared state being passed around.